Core/PSPLoaders.cpp

```diff
@@ -355,3 +356,11 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 
 	return __KernelLoadExec(finalName.c_str(), 0, error_string);
 }
+
+bool Load_PSP_GE_Dump(FileLoader *fileLoader, std::string *error_string) {
+	BlobFileSystem *umd = new BlobFileSystem(&pspFileSystem, fileLoader, "data.ppdmp");
+	pspFileSystem.Mount("disc0:", umd);
```

Comment:
Oh, and also because this let me use a `fileLoader`, which meant not having to upload to mobile devices... although, that's not really a big deal.

-[Unknown]